LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
```

Comment:
the whitespace is significant, please leave the spaces as you see them in https://raw.githubusercontent.com/hoodiehq/hoodie/master/LICENSE